src/main/java/net/sf/jsqlparser/statement/select/GroupByElement.java

```diff
@@ -126,4 +131,17 @@ public GroupByElement addGroupingSets(Collection<? extends Object> groupingSets)
         collection.addAll(groupingSets);
         return this.withGroupingSets(collection);
     }
+
+    public boolean isMysqlWithRollup() {
+        return mysqlWithRollup;
+    }
+
+    public void setMysqlWithRollup(boolean mysqlWithRollup) {
+        this.mysqlWithRollup = mysqlWithRollup;
+    }
+
+    public GroupByElement withMysqlWithRollup(boolean mysqlWithRollup) {
```

Comment:
I know, that previously those "withMethods" have been implemented, but nobody ever gave me a reason why the "setMethod" should not return the object directly.

Feel free to avoid that redundant "withMethod" and return the object when calling the "set" method.

Reply:
To my knowledge, in functional programming, it is generally referred to as with. The true meaning is to create a new object and return it without modifying the object itself, but here, with modifies the object, so set and with are indeed duplicates at this point.